.github/workflows/CICD.yml

```
@@ -122,6 +122,8 @@ jobs:
     - name: Install/setup prerequisites
       shell: bash
       run: |
+        # Avoid no space left
+        sudo rm -rf /usr/local/lib/android /usr/share/dotnet /usr/share/swift /usr/lib/firefox /opt/google/chrome &
```

```suggestion
        sudo rm -rf /usr/local/lib/android /usr/share/dotnet /usr/share/swift /usr/lib/firefox /opt/google/chrome || true
```
Ensure completion.`&` makes it async, but if it doesn't finish before disk fills, still fails.

---

or dotnet only? https://github.com/ublue-os/remove-unwanted-software/blob/master/README.md

---

My Recommendation hybrid
```yaml
sudo rm -rf /usr/local/lib/android   
sudo rm -rf /usr/share/dotnet  


sudo rm -rf /usr/share/swift /usr/lib/firefox /opt/google/chrome &

sudo apt-get -y update
sudo apt-get -y install gcc-aarch64-linux-gnu libselinux1-dev
```
If you prefer to keep full background execution:
```yaml
# Delete in background
sudo rm -rf /usr/local/lib/android /usr/share/dotnet /usr/share/swift /usr/lib/firefox /opt/google/chrome &
RM_PID=$!

# Continue with apt (parallel)
sudo apt-get -y update
sudo apt-get -y install gcc-aarch64-linux-gnu libselinux1-dev

# Ensure rm completes before builds start
wait $RM_PID || true
```